### ISSUE TITLE
Panel view "gap" is inappropriate with "Kiosk Mode" HACS plugin

### ISSUE THREAD

**Issue by Aggro600** [dermotduffy/frigate-hass-card#1762]
When I see the Frigate Card in the panel view ( 1 card on fullscreen ). As soon as I enable kiosk mode (hide_header: true + hide_sidebar: true) I have a white border under the card. This is as big as the header bar that was hidden by kiosk mode.

![Lücke](https://github.com/user-attachments/assets/972e51c6-7b3a-4ff6-86a3-4662a3be54a0)

**Comment by dermotduffy** [comment]
This is almost certainly related to https://github.com/dermotduffy/frigate-hass-card/issues/1692

The card is now leaving enough space for the header, but I think you are turning the header off entirely and so the gap is left.

> As soon as I enable kiosk mode 

What is kiosk mode? How does this get enabled?

**Comment by dermotduffy** [comment]
@Aggro600 Can you clarify what "kiosk mode" you are referring to?

**Comment by Aggro600** [comment]
You can add Kiosk Mode throught Browser Mod HACS AddOn

**Comment by dermotduffy** [comment]
Ah, not stock Home Assistant. OK, thank you.

**Comment by mckochan** [comment]
+1 to this, also another user is reporting black bar when casting to google devices. 

The previous problem could be fixed with cardmod, but I don't think this hard coded white space can be removed with card mod?

**Comment by dermotduffy** [comment]
> +1 to this, also another user is reporting black bar when casting to google devices.

@mckochan To confirm you're using browser mod / Kiosk mode?

**Comment by mckochan** [comment]
Yes, I am hiding the header on a tablet so the frigate card is truly full screen.

**Comment by tchris227** [comment]
I'm having the same issue while using kiosk mode.  #1738 subtracts `var(--header-height)` from the height.  Might it be possible to revise that to subtract the header height iff the header is visible?

**Comment by dermotduffy** [comment]
I'll install `browser_mod` to start looking into this, but could do with some reproduction instructions.

I understand I need to have "Hide Header" turned on. Does the card need to be in its own fullscreen (i.e. click the fullscreen button)? Does the card need to be casted? Do I need to use any particular type of browser?

**Comment by dermotduffy** [comment]
I ask since it seems to be working okay for me. I made a new card in panel mode, have the header turned off in Browser mod, and the card seems to appear in legitimate "fullscreen" (without using the cards own fullscreen mode), and without the gap shown at the top of this issue.

What I am doing wrong?

**Comment by tchris227** [comment]
Thank you for looking into this!  Steps to reproduce:

1. Install [Kiosk Mode](https://github.com/maykar/kiosk-mode) from HACS
2. Create a dashboard containing only a Frigate Card using Panel layout
3. Append `?kiosk` or `?hide_header` to the end of the dashboard's URL to view in kiosk mode

**Comment by dermotduffy** [comment]
Oooh. I was using the browser mod header/sidebar hiding (which seems to work fine). I do indeed see the issue with the Kiosk Mode plugin.

**Comment by Aggro600** [comment]
thank you
